resources/assets/coffee/react/beatmapset-page/scoreboard-table.coffee

```diff
@@ -16,11 +16,38 @@
 #    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
 ###
 
-{a, div, table, tr, td, th, thead, tbody} = ReactDOMFactories
+{a, button, div, i, span, table, tr, td, th, thead, tbody} = ReactDOMFactories
 el = React.createElement
 bn = 'beatmap-scoreboard-table'
 
 BeatmapsetPage.ScoreboardTable = (props) ->
+  props.scores = props.scores.map (score) ->
```

Comment:
Don't modify `props` in a component, they're supposed to be immutable within a component.

Also consider making a separate component for the report button that takes the `score` and `mode` to render